.gitlab-ci.yml

```diff
@@ -1,12 +1,13 @@
-image: humancellatlas/dss-build-box
-# The Docker image `humancellatlas/dss-build-box` is created through a manual process from
+image: quay.io/ucsc_cgl/dss-build-box
+# The Docker image `quay.io/ucsc_cgl/dss-build-box` is created through a manual process from
```

Comment:
please update the docker image comments for pushing new images. https://github.com/DataBiosphere/data-store/blob/master/allspark.Dockerfile

Reply:
Aye, good catch.